pilot/pkg/config/kube/gateway/inferencepool_collection.go

```diff
@@ -507,9 +507,10 @@ func translateShadowServiceToService(existingLabels map[string]string, shadow sh
 	ports := make([]corev1.ServicePort, 0, len(shadow.targetPorts))
 	dummyPort := int32(54320) // Dummy port, not used for anything
 	for i, port := range shadow.targetPorts {
+		portBump := int32(i) + 1
```

Comment:
maybe a dum question, how is this fixing it? isnt `servicePort.Port` always `54320+i` ?